glide.yaml

```diff
@@ -25,6 +25,7 @@ import:
   - libmachine/state
   - libmachine/swarm
 - package: github.com/google/go-github
+  version: 30a21ee1a3839fb4a408efe331f226b73faac379
```

Comment:
There is no released version nor tag. What I don't understand is the relation to #893. AFAICT, you are explicitly setting now the version as it was determined by glide.lock. So there is no update here. Would the plan for #893 not be to actually update to the latest version of go-github, which would also require some code changes? Do you want to address this as part of a dedicated pull request for #893?